platforms/dotnet/versions/bullseye/versionsToBuild.txt

```diff
@@ -0,0 +1,15 @@
+# version, sha, download url
```

Comment:
An open question that we could discuss: 
Which way we should add these debian specific txt files, adding debian folders or putting them in a single folder but naming these like this: `defaultVersions-bullseye.txt`, `defaultVersions.bullseye.txt`.

Reply:
@qianz2  I did try doing it this way, however to me it felt really cluttered. There were a whole bunch of similarly named files in the same directory. 

I can make this change, but it would be a bit of work on my end so I would like to have a great reason to do so!